### ISSUE TITLE
Overspawning of ore.

### ISSUE THREAD

**Issue by AnarchySage** [SonarSonic/Practical-Logistics#86]
The sapphire ore is everywhere, should add a config option for rarity. It spawns so much, players on my server skip iron and diamond tools for it(not your tools but it can make other sapphire mod tools)

**Comment by AnarchySage** [comment]
Alternative method, lower durability of tools

**Comment by AEnterprise** [comment]
can you provide some numbers to quantify what 'everywhere' is and how it compares to vanilla ores?

**Comment by AnarchySage** [comment]
I used a ender quarry of 64x64x64 and got 15 stacks of sapphires, however iron ore, i got 22 stacks, its nearly as common as iron, its excessive, no reason to not have a config(thus the pull request), server admin/coder/player opinion here.

**Comment by AEnterprise** [comment]
i'm not saying that having the option is a bad thing but if it's ineed to common the default value might need to be turned down but that's for sonarsonic to decide, some exact numbers always help for that

**Comment by AnarchySage** [comment]
Thats true, but as a modder side here, more config to help players/admins tweak is better than say having to fork a mod to fix what they dont like/use another mod to tweak it, etc. More configuration the better.

**Comment by AEnterprise** [comment]
i know, i'm a modder myself but changing default oregen should be kept for between MC versions as it causes confusion with pre-generated chunks, and often i had user say "there is to much" but when you look at the numbers it's realy not

**Comment by AnarchySage** [comment]
True, i reverted the config to normal values, but allows the user to customize it still. I know too much(played around with world gen in my mod(in that quarry, i got 245 stacks of one of my blocks, and 149 of the other(aesthetic blocks)))

**Comment by AnarchySage** [comment]
also keep in mind that was 15 stacks of ores(silk touch enhancement)
